lib/test_prof/factory_prof.rb

```diff
@@ -1,13 +1,19 @@
 # frozen_string_literal: true
 
 require "test_prof/factory_prof/factory_girl_patch"
```

Comment:
Let's move patches requires to the corresponding factory_builders, since we apply them there.